client/packages.go

```diff
@@ -72,3 +72,16 @@ func (client *Client) Packages() (map[string]Package, error) {
 
 	return packages, nil
 }
+
+// Package returns the most recently published revision of the package with the
+// provided name.
+func (client *Client) Package(name string) (*Package, error) {
+	var pkg *Package
+
+	path := fmt.Sprintf("/1.0/packages/%s", name)
+	if err := client.doSync("GET", path, nil, &pkg); err != nil {
+		return &Package{}, fmt.Errorf("cannot retrieve package: %s", err)
```

Comment:
Good spot, this was a casualty of changing the return type from `Package` to `*Package`. Amended.